addons/web/static/src/views/form/form_compiler.js

```diff
@@ -370,28 +370,30 @@ export class FormCompiler extends ViewCompiler {
                     mainSlot.setAttribute("itemSpan", `${itemSpan}`);
                 }
 
-                const groupClassExpr = `scope && scope.className`;
```

Comment:
in the commit message (first commit), you do not say which/what kind of class we do not need to apply anymore, nor why we don't. So looking at the diff, I see that we stop applying some class on inner groups, but I'm not sure to know if we should or not

Reply:
Also, it is no longer necessary, but not an issue, or is it causing troubles? This is 16, so ideally only bug fixes..

Reply:
> Also, it is no longer necessary, but not an issue, or is it causing troubles? This is 16, so ideally only bug fixes..
lol, yes indeed we should not make that change in 16.0, I keep forgetting that it is stable :) .

No test asserts that innergroups should give their children some specific class through the slot scope (at least not during investigation)
The innergroups template doesn't pass any scope to its slots, and since the new form grid design is younger than the compiler, I think removing them in master is safe.